build.gradle

```diff
@@ -22,6 +22,14 @@ subprojects { Project subproject ->
     }
 
     apply plugin: "org.gradle.test-retry"
+
+    dependencies {
+        api "io.micronaut.reactor:micronaut-reactor"
+
+        // TODO: Remove after completing upgrade to Micronaut 3.
+        compileOnly "jakarta.inject:jakarta.inject-api:2.0.0"
```

Comment:
This is not necessary because Micronaut 3.0.0-M2 already includes the switch to Jakarta annotations.